click/_termui_impl.py

```diff
@@ -245,7 +245,16 @@ def make_step(self, n_steps):
             return
 
         self.last_eta = time.time()
-        self.avg = self.avg[-6:] + [-(self.start - time.time()) / (self.pos)]
+
+        # self.avg is a rolling list of length <= 7 of steps where steps are
+        # defined as time elapsed divided by the total progress through
+        # self.length.
+        if not self.pos == 0:
```

Comment:
`if not foo == bar` is bad. `if self.pos != 0` is better, but you can just do `if self.pos`.

Reply:
Fixed ;)